examples/components/WithResponsiveProps/withResponsiveProps.test.js

```diff
@@ -1,6 +1,6 @@
-describe('useResponsiveComponent', () => {
+describe('withResponsiveProps', () => {
   before(() => {
-    cy.loadStory(['hooks'], ['useresponsivecomponent'])
+    cy.loadStory(['components', 'withResponsiveProps'], ['default-behavior'])
```

Comment:
I believe this should be:

```js
['components', 'withresponsiveprops'], ['default-behavior']
```

Notice that the second member is all lowercase. Sorry for such non-intuitive setup, yet it's how Storybook serializes story names. Once you correct this the tests should pass.

Reply:
yep, thanks for the heads-up 👍